visual_behavior/data_access/loading.py

```diff
@@ -1143,7 +1143,7 @@ def get_stim_metrics_summary(behavior_session_id, load_location='from_file'):
 
 # FROM LIMS DATABASE
 # this portion is depreciated, please use functions in from_lims.py instead
-gen_depr_str = 'this function is deprecated and will be removed in a future version, ' \
+gen_depr_str = 'this function is deprecated and will be removed on October 29th 2021, ' \
```

Comment:
why is this function deprecated? just trying to catch up. thanks.

Reply:
All the functions with this depreciation warning have been moved to the from_lims module of loading. So they're still accessible there, but are being removed from the more general loading.py file.